### PR TITLE
Added `request-aws-iam-resources.md` as ISSUE TEMPLATE

### PR DESCRIPTION
**Pull request by freaky4wrld**
Fixes: #10 

Provided a standardised issue-template for volunteers to request access for AWS IAM resources. Won't able to test it, open to feedbacks. Thanks

### CODE REVIEW

**Comment by chelseybeck** [comment]
@freaky4wrld thanks so much for picking this issue up! This file format should be .yml - apologies...the issue has .md but for forms it should be .yml

File: .github/ISSUE_TEMPLATE/request-aws-iam-resources.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      id: access-reason
+      attributes:
+        label: Reason for access
+        description: Mention the reason for requesting the access
+      validations:
+        required: true
+    - type: textarea
+      id: additional-info
+      attributes: 
+        labels: Addtional Information
```
Comment (chelseybeck) [inline .github/ISSUE_TEMPLATE/request-aws-iam-resources.md:31]:
```suggestion
        label: Additional Information
```

File: .github/ISSUE_TEMPLATE/request-aws-iam-resources.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: 'Request AWS IAM resources'
+description: 'Issue for requesting IAM accounts, groups, roles or policies'
```
Comment (chelseybeck) [inline .github/ISSUE_TEMPLATE/request-aws-iam-resources.md:2]:
```suggestion
description: 'Issue for requesting IAM accounts'
```
I think we should have separate issues for groups, roles, and policies...just b/c the info we need will be different

File: .github/ISSUE_TEMPLATE/request-aws-iam-resources.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: 'Request AWS IAM resources'
```
Comment (chelseybeck) [inline .github/ISSUE_TEMPLATE/request-aws-iam-resources.md:1]:
```suggestion
name: 'Request AWS IAM user or service account'
```

File: .github/ISSUE_TEMPLATE/request-aws-iam-resources.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+description: 'Issue for requesting IAM accounts, groups, roles or policies'
+title: 'Request AWS IAM resources'
```
Comment (chelseybeck) [inline .github/ISSUE_TEMPLATE/request-aws-iam-resources.md:3]:
```suggestion
```
title is not necessary as name represents the title :)

File: .github/ISSUE_TEMPLATE/request-aws-iam-resources.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+body: 
+    - type: input
+      id: account-name
+      attributes:
+        label: Account Name
+        description: Provide github handle of new user or service account name (if requesting for a service)
+      validations: 
+        required: true
+    - type: textarea
```
Comment (chelseybeck) [inline .github/ISSUE_TEMPLATE/request-aws-iam-resources.md:14]:
```suggestion
    - type: input
```
This can be shorter/doesn't need to be a textarea

**Comment by chelseybeck** [comment]
@freaky4wrld w/ all of the requested changes, here's what it looks like: https://github.com/hackforla/devops-security/blob/testing-issue-template/.github/ISSUE_TEMPLATE/request-aws-iam-resources.yml

Great work!

**Comment by freaky4wrld** [comment]
@chelseybeck after making the requested changes, it did look same the one you specified above..... thanks for your feedback..... ping me if further changes has to be made!!!

**Review by chelseybeck — APPROVED** [review]
These look great, thanks!

**Review by brittanyms — APPROVED** [review]

**Pull request merged**